syntax = "proto3";

package budlab.beamer;

message BeamerResult {
  double emittance_h = 1;
  double emittance_v = 2;
  double dependence = 3;
}